frontend/src/components/cluster-dashboard/ClusterDashboard.scss

```diff
@@ -4,4 +4,52 @@
 .home-body {
   margin-bottom: $footer-height !important;
   padding: 0px 0px $single-spacing 0px;
+}
+
+.auto-refresh-wrapper {
+  position: absolute;
+  top: 0px;
+  right: $double-spacing;
+  height: $header-height;
+  padding: $single-spacing 0px;
+
+  & .MuiTypography-root {
```

Comment:
The extra ampersand and space at the beginning of this selector can be removed. The ampersand is already implied. 

More details at:
https://css-tricks.com/the-sass-ampersand/

Reply:
@mikemunsie TIL